jira/.clasp.json

```diff
@@ -0,0 +1 @@
+{"scriptId":"1P9xa2N9H6o3DIQwUACCc_tkQnicE5L5MSTOtWp_YonePETjggs4F50w-","rootDir":"src"}
```

Comment:
Please remove this file (feel free to add it to your global .gitignore, though).

Reply:
Thank you for the feedback!